shell.nix

```diff
@@ -15,8 +15,8 @@ mkShell {
   shellHook = ''
     rm -f conductor-config.toml
     rm -f conductor-config-extra.toml
-    ln -s ${project.hp-admin-conductor-config} conductor-config.toml
-    ln -s ${project.hp-admin-extra-conductor-config} conductor-config-extra.toml
+    ln -s ${project.hp-admin-conductor-config-1} conductor-config-1.toml
+    ln -s ${project.hp-admin-conductor-config-2} conductor-config-2.toml
```

Comment:
For some reason these two lines complain when I run `nix-shell` the second and subsequent times

```
ln: failed to create symbolic link 'conductor-config-1.toml': File exists
ln: failed to create symbolic link 'conductor-config-2.toml': File exists
```

Even though all I changed was the file names and it doesn't complain in the current setup. Either of you have any ideas where I went wrong with this?

Reply:
The above suggestion should help fix this issue

Reply:
@robbiecarlton - Try running `realpath /path/file for both to see where each file lives.  

After locating them, delete those files, exit nix-shell, delete the existing conductor-configs and then re-enter into nix to trigger the nix setup with your desired updates.

Reply:
@zo-el I don't see the suggestion